Lib/test/test_doctest.py

```diff
@@ -3218,19 +3218,8 @@ def load_tests(loader, tests, pattern):
     return tests
 
 
-def test_coverage(coverdir):
-    trace = import_helper.import_module('trace')
-    tracer = trace.Trace(ignoredirs=[sys.base_prefix, sys.base_exec_prefix,],
-                         trace=0, count=1)
-    tracer.run('test_main()')
-    r = tracer.results()
-    print('Writing coverage results...')
-    r.write_results(show_missing=True, summary=True,
-                    coverdir=coverdir)
-
-
 if __name__ == '__main__':
-    if '-c' in sys.argv:
-        test_coverage('/tmp/doctest.cover')
-    else:
-        unittest.main()
+    raise RuntimeError(
+        "Running `test_doctest` via cmdline is not supported, "
+        "use `-m test test_doctest` instead"
+    )
```

Comment:
```suggestion
    unittest.main(module='test.test_doctest')
```

With this I have working ``./python Lib/test/test_doctest.py`` on main.

Reply:
This is great! Thanks! I didn't know about this option.